examples/docker/03_dind_rootless/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM ubuntu:latest
+## Install prerequisites
+RUN apt-get update && \
+  apt-get install -y apt-transport-https curl iproute2 uidmap
+# Install Docker daemon
+RUN curl -fsSL https://get.docker.com/ | sh -s -
+# Add the ubuntu user to the Docker group
+RUN usermod -aG docker ubuntu
+# Add our custom entrypoint
+ADD entrypoint.sh /entrypoint.sh
+RUN chmod +x /entrypoint.sh
```

Comment:
Git supports executable bit, so we could skip this. We also don't have it in DinD.